.github/workflows/daily.yml

```
@@ -86,11 +82,28 @@ jobs:
       id: counts
       shell: bash
       run: |
-        mc config host add S3 ${S3_HOST} ${S3_ACCESS_KEY} ${S3_SECRET_KEY}
-        mc find --newer-than ${TIME} --print '{}' ${ROOT}${LAST_CAMPAIGN} | tee -a files.txt
+        # Get the current date and the date 1 day ago
+        current_date=$(date +%s)
+        one_day_ago=$(date -d "1 day ago" +%s)
+
+        # List all files with details
+        xrdfs $XROOTD_HOST ls -l -R $ROOT | while read -r line; do
+          # Extract the date and path from the line
+          file_date=$(echo $line | awk '{print $5" "$6}')
+          file_path=$(echo $line | awk '{print $7}')
+
+          # Convert file date to seconds since epoch
+          file_date_seconds=$(date -d "$file_date" +%s)
+
+          # Check if the file was created within the last 1 day
+          if [ $file_date_seconds -ge $one_day_ago ]; then
+            echo $file_path
+          fi
+        done > files.txt
+
         # keep only the RECO/.../ path up to parent of directory with files
         cat files.txt | while read file ; do
-          if [[ $file =~ S3/eictest/EPIC/(.*)/[^/]*/[^/]* ]] ; then
+          if [[ $file =~ /work/eic2/EPIC/(.*)/[^/]*/[^/]* ]] ; then
             echo ${BASH_REMATCH[1]}
           fi
```

```suggestion
          if [[ $file =~ ${ROOT}/[^/]*/[^/]* ]] ; then
            echo ${BASH_REMATCH[0]}
          fi
```

---

Nevermind, this is not right.

---

```suggestion
          if [[ $file =~ ${ROOT}/(.*)/[^/]*/[^/]* ]] ; then
            echo ${BASH_REMATCH[1]}
          fi
```
This is better: it will also strip the RECO/ from the paths in firehose, though. I think that's fine.